configure.ac

```diff
@@ -611,6 +650,7 @@ echo
 
 echo "Building text frontend ........... : $want_textui"
 echo "Building irssi bot ............... : $want_irssibot"
+echo "Building irssi fuzzer ............ : $want_irssifuzzer"
```

Comment:
I think this is a slightly internal thing, so I don't think we should "confuse" the average user by making them wonder what "irssi fuzzer" is if they give compiling irssi for themselves a try.